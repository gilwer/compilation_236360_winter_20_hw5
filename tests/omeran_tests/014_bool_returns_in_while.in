int bar(){
    print("bar");
    return 7;
}


void main(){
    int i;
    while (bar() > i){
        print("Giri");
        i = i + 1;
    }
}