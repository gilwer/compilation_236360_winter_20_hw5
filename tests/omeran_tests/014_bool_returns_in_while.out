bar
Giri
bar
Giri
bar
Giri
bar
Giri
bar
Giri
bar
Giri
bar
Giri
bar
